easybuild/easyconfigs/m/MEME/MEME-5.4.1-GCC-11.2.0.eb

```
@@ -0,0 +1,39 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'MEME'
+version = '5.4.1'
+
+homepage = 'https://meme-suite.org/meme/index.html'
+description = """The MEME Suite allows you to: * discover motifs using MEME, DREME (DNA only) or
+ GLAM2 on groups of related DNA or protein sequences, * search sequence databases with motifs using
+ MAST, FIMO, MCAST or GLAM2SCAN, * compare a motif to all motifs in a database of motifs, * associate
+ motifs with Gene Ontology terms via their putative target genes, and * analyse motif enrichment
+ using SpaMo or CentriMo."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://%(namelower)s-suite.org/%(namelower)s/%(namelower)s-software/%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['c07fb8afafa60fc5e84ca24493c82fa6f4bd1df1a2622102edbf86a1c30fd11f']
+
+dependencies = [
+    ('libxml2', '2.9.10'),
+    ('libxslt', '1.1.34'),
+    ('zlib', '1.2.11'),
+    ('Perl', '5.34.0'),
+    ('Python', '3.9.6')
+]
+
+configopts = '--enable-build-libxml2 --enable-build-libxslt '
+configopts += '--with-perl=${EBROOTPERL}/bin/perl --with-python3=${EBROOTPYTHON}/bin/python '
+
+sanity_check_paths = {
+    'files': ['bin/dreme', 'bin/meme', 'bin/dreme', 'bin/meme-chip'],
+    'dirs': ['lib'],
+}
+
```

should probably add `libexec` to `PATH`, and probably ensure the perl modules are properly picked up as well
```suggestion

modextrapaths = {
    'PATH': ['libexec'],
    'PERL5LIB': ['lib/%(namelower)s-%(version)s/perl'],
}

```

---

fixed in [752d645](https://github.com/easybuilders/easybuild-easyconfigs/pull/15169/commits/752d6454b6437d018cc0c59b1224c59690538985)